src/pages/App.jsx

```diff
@@ -17,6 +18,8 @@ const App = () => (
       <Route path="/signup" component={Register}></Route>
       <Route path="/profile" exact component={Profile}></Route>
       <Route path="/forgot-password" component={ForgotPassword}></Route>
+      {/* <Route path="/dashboard" exact component={Index}></Route> */}
```

Comment:
Please why the comment?

Reply:
it is an oversight, I have removed it now thanks for spotting it